### DISCUSSION TITLE
Vis.js Network Map Element

### DISCUSSION THREAD

**Discussion by Ranger11Danger** [zauberzeug/nicegui#1682] [Ideas / Feature Requests]
i have a working example on using the vis.js library to create network maps with vis.js and was wondering if theres any appetite for me add the example to the docs or try and implement it as built in element? it follows the same principles that the leaflesjs example uses.

**Comment by falkoschindler** [comment]
Sounds very interesting, @Ranger11Danger!
Would you like to create a PR with an example? From there we can decide about a possible integration into NiceGUI. 🙂

**Reply by Ranger11Danger** [reply]
@falkoschindler sure I’ll start working on an example, would just following the structure of the leaflet example be good?

**Reply by falkoschindler** [reply]
@Ranger11Danger Yes, I think this would be a good starting point. 🙂
